src/udev/udev-rules.c

```diff
@@ -1152,13 +1159,15 @@ static void rule_resolve_goto(UdevRuleFile *rule_file) {
                 if (!line->goto_line) {
                         log_error("%s:%u: GOTO=\"%s\" has no matching label, ignoring",
                                   rule_file->filename, line->line_number, line->goto_label);
+                        rule_file_mark_issue(rule_file, LOG_ERR);
```

Comment:
Maybe, better to introduce `log_line_error()` and friends that takes `UdevRuleLine` object.